templates/linux-template.j2

```diff
@@ -10,6 +10,7 @@ metadata:
   labels:
     kubevirt.io/os: rhel7.4
     miq.github.io/kubevirt-is-vm-template: "true"
+    import-vm-apb/transaction_id: "{{ _apb_service_instance_id }}"
```

Comment:
You want to label the template itself, not the vms it will create as separate actions later.

Reply:
I am, the VM definition is below.